mkosi/__init__.py

```diff
@@ -6610,6 +6616,11 @@ def setup_ssh(
     if args.ssh_key:
         f = open(args.ssh_key, mode="r", encoding="utf-8")
         copy_file(f"{args.ssh_key}.pub", authorized_keys)
+    elif args.ssh_agent_socket is not None:
+        env = {"SSH_AUTH_SOCK": args.ssh_agent_socket}
+        result = run(["ssh-add", "-L"], env=env, check=True, text=True, stdout=subprocess.PIPE)
```

Comment:
`check=True` is the default if I remember correctly so the `check=True` param can be removed

Reply:
I'm not sure, I used it because it's used in many places (search for check=True)

Reply:
Yeah, just check the definition of run() in backend.py, it's the default. I'll do a commit to fix the other callsites

Reply:
ok, removed